Tests/TestMessagesViewControllerModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TestMessagesViewControllerModel.swift
+//  MessageKit
+//
+//  Created by Dan Leonard on 8/14/17.
+//  Copyright © 2017 MessageKit. All rights reserved.
+//
```

Comment:
This is possible as of Xcode 9, nice article explaining it [here](https://oleb.net/blog/2017/07/xcode-9-text-macros/)!